plugin/vcs/vcs.go

```diff
@@ -147,18 +160,22 @@ type Provider interface {
 	//
 	// Similar to CreateFile except it overwrites an existing file. The fileCommit shoud includes the "LastCommitID" field which is used to detect conflicting writes.
 	OverwriteFile(ctx context.Context, oauthCtx common.OauthContext, instanceURL, repositoryID, filePath string, fileCommit FileCommitCreate) error
-	// Reads the file content. Returns an io.ReadCloser on success. If file does not exist, returns NotFound error.
+	// Reads the file metadata
 	//
-	// oauthCtx: OAuth context to read the file content
+	// oauthCtx: OAuth context to fetch the file data
 	// instanceURL: VCS instance URL
 	// repositoryID: the repository ID from the external VCS system (note this is NOT the ID of Bytebase's own repository resource)
 	// filePath: file path to be read
-	// commitID: the specific version to be read
-	ReadFile(ctx context.Context, oauthCtx common.OauthContext, instanceURL, repositoryID, filePath, commitID string) (string, error)
-	// Reads the file metadata. Returns the file meta on success.
+	// ref: the name of branch, tag or commit
+	ReadFileMeta(ctx context.Context, oauthCtx common.OauthContext, instanceURL, repositoryID, filePath, ref string) (*FileMeta, error)
+	// Reads the file content by commit id. Returns an io.ReadCloser on success. If file does not exist, returns NotFound error.
```

Comment:
nit: here says by commit id, but the ref comment say something else.

And let's be consistent between the ReadFileMata and ReadFileContent comment